subprojects/testing-jvm/src/main/java/org/gradle/api/tasks/testing/Test.java

```diff
@@ -956,75 +960,95 @@ TestFramework useTestFramework(TestFramework testFramework) {
         useJUnit(ConfigureUtil.<JUnitOptions>configureUsing(testFrameworkConfigure));
     }
 
     /**
-     * Specifies that JUnit should be used to execute the tests, configuring JUnit specific options. <p> The supplied action configures an instance of {@link
-     * org.gradle.api.tasks.testing.junit.JUnitOptions}, which can be used to configure how JUnit runs.
+     * Specifies that JUnit4 should be used to discover and execute the tests with additional configuration.
+     * <p>
+     * The supplied action configures an instance of {@link org.gradle.api.tasks.testing.junit.JUnitOptions JUnit4 specific options}.
      *
-     * @param testFrameworkConfigure An action used to configure the JUnit options.
+     * @param testFrameworkConfigure An action used to configure JUnit4 options.
      * @since 3.5
      */
     public void useJUnit(Action<? super JUnitOptions> testFrameworkConfigure) {
         useTestFramework(new JUnitTestFramework(this, (DefaultTestFilter) getFilter()), testFrameworkConfigure);
     }
 
     /**
-     * Specifies that JUnit Platform (a.k.a. JUnit 5) should be used to execute the tests. <p> To configure JUnit platform specific options, see {@link #useJUnitPlatform(Action)}.
+     * Specifies that JUnit Platform should be used to discover and execute the tests.
+     * <p><br>
+     * Use this option if your tests use JUnit Jupiter/JUnit5.
+     * <p><br>
+     * JUnit Platform supports multiple test engines, which allows other testing frameworks to be built on top of it.
+     * You may need to use this option even if you are not using JUnit directly.
      *
      * @since 4.6
+     * @see #useJUnitPlatform(org.gradle.api.Action) Configure JUnit Platform specific options.
      */
     public void useJUnitPlatform() {
         useJUnitPlatform(Actions.<JUnitPlatformOptions>doNothing());
     }
 
     /**
-     * Specifies that JUnit Platform (a.k.a. JUnit 5) should be used to execute the tests, configuring JUnit platform specific options. <p> The supplied action configures an instance of {@link
-     * org.gradle.api.tasks.testing.junitplatform.JUnitPlatformOptions}, which can be used to configure how JUnit platform runs.
+     * Specifies that JUnit Platform should be used to discover and execute the tests with additional configuration.
+     * <p><br>
+     * Use this option if your tests use JUnit Jupiter/JUnit5.
+     * <p><br>
+     * JUnit Platform supports multiple test engines, which allows other testing frameworks to be built on top of it.
+     * You may need to use this option even if you are not using JUnit directly.
+     * <p>
+     * The supplied action configures an instance of {@link org.gradle.api.tasks.testing.junitplatform.JUnitPlatformOptions JUnit Platform specific options}.
      *
-     * @param testFrameworkConfigure An action used to configure the JUnit platform options.
+     * @param testFrameworkConfigure A closure used to configure JUnit platform options.
```

Comment:
Is it correct/convention to use `closure` as an abstract term here? Could this mistakenly be associated with `groovy.lang.Closure`?

Reply:
ah, I meant to use action here. Thanks for noticing